bin/inasafe

```diff
@@ -0,0 +1,318 @@
+#!/usr/bin/python
+# coding=utf-8
+"""
+InaSAFE Disaster risk assessment tool developed by AusAid -
+**CLI implementation for inasafe project.**
+
+Contact : jannes@kartoza.com
+
+.. note:: This program is free software; you can redistribute it and/or modify
+     it under the terms of the GNU General Public License as published by
+     the Free Software Foundation; either version 2 of the License, or
+     (at your option) any later version.
+
+    Note :
+        cli: command line interface
+"""
+
+__author__ = 'Jannes Engelbrecht'
+__date__ = '16/04/15'
+from docopt import docopt, DocoptExit
+from safe.impact_functions.registry import Registry
+from safe.impact_functions import register_impact_functions
+from safe.test.utilities import get_qgis_app
+from safe.impact_functions.impact_function_manager import ImpactFunctionManager
+from qgis.core import (
+    QgsRasterLayer,
+    QgsVectorLayer,
+    QgsRectangle,
+    QgsMapLayerRegistry,
+    QgsCoordinateReferenceSystem)
+from safe.utilities.keyword_io import KeywordIO
+from safe.utilities.gis import qgis_version
+import os
+from safe.report.impact_report import ImpactReport
+from PyQt4.QtCore import QSettings
+import logging
+
+current_dir = os.path.abspath(
+    os.path.realpath(os.environ['PWD']))
+usage_dir = os.environ['InaSAFEQGIS'] + '/bin'
+usage = r""
+usage_file = file(os.path.join(usage_dir, 'usage.txt'))
+for delta in usage_file:
+    usage += delta
+LOGGER = logging.getLogger('InaSAFE')
+QGIS_APP, CANVAS, IFACE, PARENT = get_qgis_app()
+
+
+class CliArgs(object):
```

Comment:
Is this naming convention required by docopts? If not please call CommandLineArguments or similar (to follow our coding conventions).